frontend/src/libs/components/select/select.tsx

```diff
@@ -18,18 +18,62 @@ type Properties<T extends FieldValues> = {
   options: SelectOption<string | number>[];
   placeholder?: string;
   errors: FieldErrors<T>;
+  isMulti?: boolean;
+  hideSelectedOptions: boolean;
+  customComponents?: object;
```

Comment:
First of all custom components in react-select may be typed,, and the second - you should provide customComponents as outer props - but just specify and pass needed ones if it's multiselect